ci-operator/config/openshift/kube-proxy/openshift-kube-proxy-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.1"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+canonical_go_repository: github.com/openshift/origin
+images:
+- from: hyperkube
```

Comment:
`hyperkube` is unknown here, I believe you would need to reference this in `base_images` to be able to build from it.